pkg/appmanager/resourceConfig.go

```diff
@@ -577,15 +577,19 @@ func copyConfigMap(cfg *ResourceConfig, cfgMap *ConfigMap) {
 			name = fmt.Sprintf("%s", cfg.Virtual.VirtualServerName)
 		}
 		monitor := Monitor{
-			Name:      name,
+			// Append the protocol to the monitor names to differentiate them.
+			// Also add a monitor index to the name to be consistent with the
+			// marathon-bigip-ctlr. Since the monitor names are already unique here,
+			// appending a '0' is sufficient.
+			Name:      name + "_0_" + mon.Protocol,
```

Comment:
I understand the need to add the protocol to the end here, but it seems like the "0" shouldn't be added here; instead the "if index > 0" block above changed to just format the name as expected. As this is now, the first monitor might look like "name_0_http", but the second would look like "name_1_0_http" which seems wrong ("name_1_http" seems better).

Reply:
Agree